osx/install_swift_binaries.sh

```diff
@@ -33,7 +33,17 @@ brew install wget > /dev/null || brew outdated wget > /dev/null || brew upgrade
 # See http://apple.stackexchange.com/questions/72226/installing-pkg-with-terminal
 # TODO: Since Xcode now includes the Swift compiler and Xcode is included in the macOS image provided
```

Comment:
@KyeMaloy97 As part of this PR, can we remove the TODO comment from the code (see lines above)?